server/odc-service/src/main/java/com/oceanbase/odc/service/onlineschemachange/oms/openapi/ProjectOpenApiService.java

```diff
@@ -30,6 +32,15 @@
  * @since 4.2.0
  */
 public interface ProjectOpenApiService {
```

Comment:
Ditto.

Reply:
this class suggest reserve, it's referenced by many class and name contain `OpenApi` which can be different from odc project

Reply:
done , all osc class with project* rename to OmsProject*